<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuPrincipal</class>
 <widget class="QMainWindow" name="MenuPrincipal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MenuPrincipal</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Crear Legislador:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre (Int):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombre">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>113</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CrearLeg">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Descripcion:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Crear Ley:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDesc">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>221</width>
      <height>33</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CrearLey">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>B/M:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="check">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>95</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>220</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Aprobar Ley:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ley:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbLey">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>281</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Legislador:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbLegislador1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>231</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AprobarLey">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Aprobar</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbConjunto">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>231</width>
      <height>29</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Buenos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Malos</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="ComprobarMiembrp">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>96</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Comprobar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Legislador:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbLegislador2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>231</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>350</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Es Miembro?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Conjunto:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
